CoreFoundation/RunLoop.subproj/CFRunLoop.h

```diff
@@ -29,7 +29,7 @@ typedef struct CF_BRIDGED_MUTABLE_TYPE(id) __CFRunLoopSource * CFRunLoopSourceRe
 
 typedef struct CF_BRIDGED_MUTABLE_TYPE(id) __CFRunLoopObserver * CFRunLoopObserverRef;
 
-typedef struct CF_BRIDGED_MUTABLE_TYPE(NSTimer) __CFRunLoopTimer * CFRunLoopTimerRef;
+typedef struct /*CF_BRIDGED_MUTABLE_TYPE(id)*/ __CFRunLoopTimer * CFRunLoopTimerRef;
```

Comment:
Please do not disable this. This does not bridge to `NSTimer`; it bridges to `id`, meaning that it allows `CFRunLoopTimer` to be treated as an `AnyObject` (and thus its API be imported without using `Unmanaged<…>`.

Reply:
OK makes sense, I did note CFRunLoop itself etc are also  `CF_BRIDGED_MUTABLE_TYPE` there. Should the type remain `NSTimer` however or be `id` like the rest?